ids/bits_test.go

```diff
@@ -71,8 +71,8 @@ func TestEqualSubsetBadMiddle(t *testing.T) {
 }
 
 func TestEqualSubsetAll3Bytes(t *testing.T) {
-	rand.Seed(time.Now().UnixNano())
-	seed := uint64(rand.Int63()) // #nosec G404
+	rand := rand.New(rand.NewSource(time.Now().UnixNano())) //#nosec G404
+	seed := uint64(rand.Int63())                            //#nosec G404
```

Comment:
The default behavior is for the randomness to be seeded, why do we seed it again here?
```suggestion
	seed := rand.Uint64()
```